### DISCUSSION TITLE
How do you reference an already-created CentralPanel in one frame?

### DISCUSSION THREAD

**Discussion by Lertos** [emilk/egui#1341] [Q&A]
For example, I'm using Bevy and I have multiple systems. In one system I want to create the UI. In the next I would like to access the Central Panel and either change or add to the UI contents. Is this possible?

I have a feeling I may be thinking about the design all wrong; but no matter what I come up with, my "show_ui" system (the one that creates the panels) ends up being enormous.

**Comment by emilk** [comment]
I think you need to wrap your head around immediate mode a bit better: https://docs.rs/egui/latest/egui/#understanding-immediate-mode
